lib/imgix/path.rb

```diff
@@ -133,15 +133,16 @@ def build_srcset_pairs(options:, params:)
 
       widths = options[:widths] || []
       width_tolerance = options[:width_tolerance] ||  DEFAULT_WIDTH_TOLERANCE
-      min_srcset = options[:min_width] || MIN_WIDTH
-      max_srcset = options[:max_width] || MAX_WIDTH
+      min_width = options[:min_width] || MIN_WIDTH
+      max_width = options[:max_width] || MAX_WIDTH
 
       if !widths.empty?
         validate_widths!(widths)
         srcset_widths = widths
-      elsif width_tolerance != DEFAULT_WIDTH_TOLERANCE or min_srcset != MIN_WIDTH or max_srcset != MAX_WIDTH
-        validate_range!(min_srcset, max_srcset)
-        srcset_widths = TARGET_WIDTHS.call(width_tolerance, min_srcset, max_srcset)
+      elsif width_tolerance != DEFAULT_WIDTH_TOLERANCE || min_width != MIN_WIDTH or max_width != MAX_WIDTH
```

Comment:
Shouldn't this be `||` as well?
```suggestion
      elsif width_tolerance != DEFAULT_WIDTH_TOLERANCE || min_width != MIN_WIDTH || max_width != MAX_WIDTH
```

Reply:
Yes! Good catch!